docs/release-workflow.md

```diff
@@ -7,14 +7,34 @@ Reduce manually performed operation during csi release procedure.
 Use GitHub actions for release charts and update helm repository.
 
 Whole release workflow contains 2 parts:
-1. Release workflow in current repo triggered on release creation
+1. Release workflow in current repo triggered on release or pre-release creation
 2. Release workflow in [csi-baremetal-operator](https://github.com/dell/csi-baremetal-operator) repo
 
-### Detailed description
-
+## Detailed description
 ![Getting Started](./images/release_workflow.png)
 
-## Restrictions:
+## Restrictions
 * release branches in both repos should have the same names
 * release description should has the following format:
 `csi_version: <version>, csi_operator_version: <version>`
+
+## Testing strategy
+Workflows linting is performed on the [actionlint](https://github.com/rhysd/actionlint) base ([shellcheck](https://github.com/koalaman/shellcheck) is required).
+Local workflow testing can be performed with [Act](https://github.com/nektos/act).
+
+Steps to run tests:
+1. `make prepare-env` will download and install mentioned tools
+2. `make workflows-lint` will lint all workflows in `.github/workflows` dir
+3. Create file `.github/workflows/tests/wf.secrets` with GITHUB_TOKEN variable definition. Token is generated on [token page of GitHub user settings](https://github.com/settings/tokens). Required permissions: repo, workflow.
+4. Login to ghcr.io with GH username and GITHUB_TOKEN: `echo $(cat .github/workflows/tests/wf.secrets | awk -F "=" '{print $2}') | docker login ghcr.io -u USERNAME --password-stdin`
+5. `make test-release-workflow` will run workflow locally and execute all steps except marked as `${{ !env.ACT }}`
+
+Release workflow in csi-baremetal-operator workflow test is started by the following cmd:
+`make test-release-workflow`
+
+## Usage
+Create release in this repo choosing necessary tag, target, fill title and description using the following format:
```

Comment:
As I understand it, we have the following parameters to run action:
1. tag
2. branch
3. is_prerelease
4. csi/operator version

Could we find branch name and csi/operator versions by tag and reduce these parameters?

Reply:
We can't go without tag and branch, as GH won't let you sending a form without these parameters. Is_prerelease is necessary for deciding which release type should be created in operator repo, as it's the same as in this repo. As for versions - yes, we'd be able to reduce them after the next workflow improvement about test coverage. We discussed it today.